addons/l10n_in_edi_ewaybill/views/res_config_settings_views.xml

```diff
@@ -13,7 +13,7 @@
                             <t class="o_form_label">Setup E-Waybill</t>
                             <span class="fa fa-lg fa-building-o" title="Values set here are company-specific." aria-label="Values set here are company-specific." groups="base.group_multi_company" role="img"/>
                             <div class="text-muted">
-                                Setup E-Waybill Service for this company
+                                Check the <a href="https://www.odoo.com/documentation/15.0/applications/finance/accounting/fiscal_localizations/localizations/india.html">documentation</a> to get credentials
```

Comment:
should the link not be version specific

Reply:
@h4818  As I see documentation argument is used in the master.
I don't know, it's available in V15 and V16?